atest/requirements-run.txt

```diff
@@ -1 +1,2 @@
 xmlschema
+typing-extensions; python_version == '3.7'
```

Comment:
This should be in `atest/requirements.txt` or in `utest/requirements.txt` that's read by the former, not here.